soc/arm/nordic_nrf/Kconfig.defconfig

```diff
@@ -27,6 +27,9 @@ config SYS_CLOCK_TICKS_PER_SEC
 config ARCH_HAS_CUSTOM_BUSY_WAIT
 	default y
 
+config PM
```

Comment:
I don't think the right place for a board option is the soc directory, just because of the board-soc hierharhy.

Reply:
This is restoring behavior that existed for well over a year until it was removed as a side effect.  Are you sure you want to block on this?